utils/extractor/src/index.ts

```diff
@@ -293,3 +293,53 @@ export type SheetCapture = {
   data: Flatfile.RecordData[]
   metadata?: { rowHeaders: number[] }
 }
+
+export function parseSheet(jsonArray: any[]): SheetCapture {
```

Comment:
AI suggested depth limit: https://github.com/FlatFilers/flatfile-plugins/pull/693/files#r1826003570

Deferring for now as this has product implications (see deeply nested test case examples)